src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/AgencyInternalLocation.kt

```diff
@@ -16,7 +16,7 @@ import java.util.Objects
 
 @Entity
 @Table(name = "AGENCY_INTERNAL_LOCATIONS")
-data class AgencyInternalLocation(
+class AgencyInternalLocation(
   @Id
```

Comment:
a perfect attempt to  get the proxy to work in JPA - no need for data class anyway